src/codec.spec.ts

```diff
@@ -34,22 +36,22 @@ describe("RLN codec with version 0", () => {
     rlnInstance.insertMember(memKeys.IDCommitment);
 
     const rlnEncoder = new RLNEncoder(
-      new EncoderV0(TestContentTopic),
+      new Encoder(TestContentTopic),
```

Comment:
nitpick: `createEncoder` is preferred as it is the API we encourage developers to use.